src/slave/containerizer/mesos/isolators/xfs/disk.cpp

```diff
@@ -223,14 +224,16 @@ XfsDiskIsolatorProcess::XfsDiskIsolatorProcess(
     xfs::QuotaPolicy _quotaPolicy,
     const std::string& _workDir,
     const IntervalSet<prid_t>& projectIds,
-    Duration _projectWatchInterval)
+    Duration _projectWatchInterval,
+    const Bytes& _quotaHeadroom)
```

Comment:
Any reason to pass it by const reference instead of value?
I'd assume `Bytes`must be at most a word or two, and seems to be passed by value elsewhere.